schema_based_on_diagram.sql

```diff
@@ -0,0 +1,45 @@
+createdb digram_db
+
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INTEGER ,
+  medical_history_id INTEGER ,
+  status TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  contact_info VARCHAR(255)
+);
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR(255),
+  name VARCHAR(255),
+);
+CREATE TABLE invoice_items (
+  id SERIAL PRIMARY KEY,
+  unit_price DECIMAL(10,2),
+  quantity INTEGER,
+  total_price DECIMAL(10,2),
+  invoice_id INTEGER REFERENCES invoices(id),
+  treatment_id INTEGER REFERENCES treatments(id)
+);
+
+ALTER TABLE medical_histories
+ADD CONSTRAINT fk_patient_id
+FOREIGN KEY (patient_id)
+REFERENCES patients(id);
+
+ALTER TABLE medical_histories
+ADD CONSTRAINT fk_medical_history_id
+FOREIGN KEY (medical_history_id)
+REFERENCES medical_histories(id);
+
+CREATE TABLE medical_treatment(
+  medical_histories_id INTEGER REFERENCES medical_histories (id),
+  treatments_id INTEGER REFERENCES treatments(id),
+);
```

Comment:
- Kindly create INDEX for each table as it's required on the project requirements

![image](https://user-images.githubusercontent.com/93630700/229864387-e2a4dc0c-d5ec-4ce8-826b-d1ded82bd4fc.png)